From: Jan Glauber <jglauber@redhat.com>
Subject: [RHEL5 PATCH] add missing ctcmpc Makefile target
Date: Thu, 23 Nov 2006 09:45:53 +0100
Bugzilla: 184608
Message-Id: <1164271554.5596.6.camel@bender>
Changelog: add missing ctcmpc Makefile target


BZ 184608.

Allthough the ctcmpc driver isn't upstream its Makefile target was in 2.6.17.
The Makefile entry was removed in 2.6.18, therefore ctcmpc is now missing in RHEL5.

This patch adds the Makefile target, so ctcmpc is actually build.

Jan
-- 
jglauber@redhat.com
jang@de.ibm.com

--- linux-2.6.18.s390/drivers/s390/net/Makefile.ctcmpc
+++ linux-2.6.18.s390/drivers/s390/net/Makefile
@@ -8,6 +8,7 @@ obj-$(CONFIG_IUCV) += iucv.o
 obj-$(CONFIG_NETIUCV) += netiucv.o fsm.o
 obj-$(CONFIG_SMSGIUCV) += smsgiucv.o
 obj-$(CONFIG_CTC) += ctc.o fsm.o cu3088.o
+obj-$(CONFIG_MPC) += ctcmpc.o fsm.o cu3088.o
 obj-$(CONFIG_LCS) += lcs.o cu3088.o
 obj-$(CONFIG_CLAW) += claw.o cu3088.o
 qeth-y := qeth_main.o qeth_mpc.o qeth_sys.o qeth_eddp.o 

